core/src/main/java/com/netease/arctic/catalog/ArcticCatalog.java

```diff
@@ -52,6 +53,8 @@ public interface ArcticCatalog {
    */
   void initialize(AmsClient client, CatalogMeta meta, Map<String, String> properties);
 
+  void initialize(String name, Map<String, String> properties, TableMetaStore metaStore);
```

Comment:
This `initialize` method needs some docs.

Additionally, I'm a bit confused about when to use these two initialize methods and whether the properties refer to client properties or catalog properties. It would be better to differentiate them in the parameter names and docs.

Reply:
this method is deprecated.

Reply:
I have been removed this method.